app/code/community/ShipStream/Sync/Model/Cron.php

```diff
@@ -28,10 +28,21 @@ public function fullInventorySync($sleep = TRUE)
                     $db->beginTransaction();
                     try {
                         $target = $this->_getTargetInventory(array_keys($source));
+                        //Get qty of order items that are in processing state and not submitted to shipstream
+                        $processingQty = $this->_getProcessingOrderItemsQty(array_keys($source));
                         foreach ($source as $sku => $qty) {
-                            if ( ! isset($target[$sku])) continue;
-                            if (floatval($qty) === floatval($target[$sku]['qty'])) continue;
-                            Mage::log("SKU: $sku remote qty is $qty and local is {$target[$sku]['qty']}", Zend_Log::DEBUG, self::LOG_FILE);
+                            if ( ! isset($target[$sku])) {
+                                continue;
+                            }
+                            $syncQty = $qty =  floor(floatval($qty));
+                            if(isset($processingQty[$sku])) {
+                                $syncQty = floor(bcsub($qty, floatval($processingQty[$sku]['qty'], 4)));
+                            }
+                            $targetQty = floatval($target[$sku]['qty']);
+                            if ($syncQty === $targetQty) {
```

Comment:
This may work but it is safer to use bccomp with scale 0:
```suggestion
                            if (bccomp($syncQty, $targetQty, 0) === 0) {
```

Reply:
`bccomp` function is creating issues with `inventory_sync` method call. So for now I am not using this